assets/helpers/internationalisation/country.js

```diff
@@ -140,8 +255,24 @@ function setCountry(country: IsoCountry) {
   cookie.set('GU_country', country, 7);
 }
 
+function handleEuroCountry(): ?IsoCountry {
+  const path = window.location.pathname;
+
+  if (path !== '/eu' && !path.startsWith('/eu/')) {
+    return null;
+  }
+
+  const candidateCountry: ?IsoCountry = fromQueryParameter() || fromCookie() || fromGeolocation();
+
+  if (candidateCountry && isEuroCountry(candidateCountry)) {
+    return candidateCountry;
+  }
+
+  return 'DE';
```

Comment:
By default we say they're from Germany?

Reply:
Yeap, by default, if the URL starts with `/eu` and we can't determine a euro country we put Germany (our top contributor euro country)